Fix configure failure:
gtkdocize: GTK_DOC_CHECK not called in ./configure.ac
https://aur.archlinux.org/cgit/aur.git/tree/gtk-doc.patch?h=webkitgtk
--- configure.ac.orig
+++ configure.ac
@@ -36,6 +36,13 @@ AM_INIT_AUTOMAKE([foreign subdir-objects dist-xz no-dist-gzip tar-ustar])
 AM_MAINTAINER_MODE([enable])
 m4_include([Source/autotools/SetupAutomake.m4])
 
+# check for gtk-doc
+m4_ifdef([GTK_DOC_CHECK], [
+GTK_DOC_CHECK([1.14],[--flavour no-tmpl])
+],[
+AM_CONDITIONAL([ENABLE_GTK_DOC], false)
+])
+
 ######################################################################################
 # Processing of configuration files
 ######################################################################################
